e2e/cypress/support/ui/account_settings_modal.js

```diff
@@ -25,3 +25,41 @@ Cypress.Commands.add('verifyAccountNameSettings', (firstname, lastname) => {
     cy.get('#nameDesc').should('have.text', `${firstname} ${lastname}`);
     cy.uiClose();
 });
+
+Cypress.Commands.add('uiChangeGenericDisplaySetting', (setting, option) => {
+    cy.uiOpenAccountSettingsModal();
```

Comment:
I think this should be
```suggestion
    cy.uiOpenAccountSettingsModal('Display');
```